docs/fundamentals/code-analysis/style-rules/ide0003_ide0009.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "'this.' and 'Me.' qualifiers (IDE0003 and IDE0009)"
```

Comment:
Example doc page for IDE code style option group that use multiple complimentary rules, which need to be documented in a single page.

Reply:
NOTE: Almost all content here is taken from existing `language-rules.md` file.